erpnext/accounts/doctype/account_group/account_group.py

```diff
@@ -0,0 +1,67 @@
+from frappe.model.document import Document
+import frappe
+
+class AccountGroup(Document):
+
+    def validate(self):
+        self.validate_root_level()
+        self.validate_rows()
+
+    def validate_root_level(self):
+        """Validate root level account group type."""
+        if self.is_root_level:
+            if self.reporting_type not in ['Profit and Loss', 'Balance Sheet']:
+                frappe.throw('Root level account groups must be either Profit and Loss or Balance Sheet type')
+            
+            # Check if another root level group exists for this reporting type
+            existing_root = frappe.db.get_value('Account Group', 
+                {
+                    'company': self.company,
+                    'is_root_level': 1,
+                    'reporting_type': self.reporting_type,
+                    'name': ['!=', self.name] 
+                }, 
+                ['name', 'group_name'],
+                as_dict=1
+            )
+            
+            if existing_root:
+                frappe.throw(
+                    f'Another root level group "{existing_root.group_name}" already exists for {self.reporting_type} reporting type. '
+                    f'<a href="/app/account-group/{existing_root.name}">Click here to view</a>'
+                )
```

Comment:
do not use f-strings for translations, use numbered-placeholder string formatting